src/arrays.js

```diff
@@ -1,61 +1,84 @@
 const getNthElement = (index, array) => {
-  // your code here
+  // if (index === array.length
+  // return array.at(index);
+
+  // if (index > array.length - 1) {
+  //   index -= array.length;
+  // }
+  // return array[index];
+
+  return array[index % array.length];
 };
 
 const arrayToCSVString = array => {
-  // your code here
+  return array.join(',');
 };
 
 const csvStringToArray = string => {
-  // your code here
+  return Array.from(string.replace(/,/g, ''));
 };
 
 const addToArray = (element, array) => {
-  // your code here
+  array.push(element);
 };
 
 const addToArray2 = (element, array) => {
-  // your code here
+  return [...array, element];
 };
 
 const removeNthElement = (index, array) => {
-  // your code here
+  array.splice(index, 1);
 };
 
 const numbersToStrings = numbers => {
-  // your code here
+  return numbers.toString().split(',');
 };
 
 const uppercaseWordsInArray = strings => {
-  // your code here
+  return strings.map(strings2 => strings2.toUpperCase());
 };
 
 const reverseWordsInArray = strings => {
-  // your code here
+  return strings
+    .toString()
+    .split('')
+    .reverse()
+    .join('')
+    .split(',')
+    .reverse();
```

Comment:
Interesting approach here. I see what you were thinking, but there is a cleaner way to do this using the same methods:
```suggestion
  return strings.map(string => {
    return string
      .split('')
      .reverse()
      .join('');
  });
```